integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CarbonScalaUtil.scala

```diff
@@ -152,6 +154,22 @@ object CarbonScalaUtil {
     }
   }
 
+  def getString(value: String,
```

Comment:
move parameter to next line, and add comment for this function.
Maybe rename to `convertToUTF8String` is better

Reply:
ok